.github/workflows/nodejs.yml

```
@@ -119,6 +126,24 @@ jobs:
         run: npm run test:coverage -- --ci --shard=${{ matrix.shard }}
         if: matrix.node-version != '18.x'
 
+      - name: Install Playwright browsers and dependencies
+        run: npx playwright install --with-deps
+
+      - name: Set up cache
+        id: cache
+        uses: actions/cache@v4
+        with:
+          key: cache/${{github.repository}}/${{github.ref}}
+          restore-keys: cache/${{github.repository}}/refs/heads/master
+          path: .test/**
+
+      - name: Initialize snapshots
+        if: ${{steps.cache.outputs.cache-hit != 'true' || inputs.update-snapshots == 'true'}}
+        run: npx playwright test --update-snapshots
+
+      - name: Run E2E tests for webpack version ${{ matrix.webpack-version }}
+        run: npm run test:e2e
```

Remove the steps at line 121 and 125, also this should be:

```suggestion
      - name: Run E2E tests for webpack version ${{ matrix.webpack-version }}
        run: npm run test:e2e -- --shard=${{ matrix.shard }}
```

---

I set a specific folder for tests and now its working.